scripts/publish.sh

```diff
@@ -160,9 +160,13 @@ publish_to_docker_hub() {
 
 		create_manifest_list ${1} "latest" ${AWS_FOR_FLUENT_BIT_VERSION}
 		create_manifest_list ${1} ${AWS_FOR_FLUENT_BIT_VERSION} ${AWS_FOR_FLUENT_BIT_VERSION}
+		create_manifest_list_debug ${1} "debug-latest" ${AWS_FOR_FLUENT_BIT_VERSION}
+		create_manifest_list_debug ${1} "debug"-${AWS_FOR_FLUENT_BIT_VERSION} ${AWS_FOR_FLUENT_BIT_VERSION}
 
 		create_manifest_list_init ${1} "init-latest" ${AWS_FOR_FLUENT_BIT_VERSION}
 		create_manifest_list_init ${1} "$init"-${AWS_FOR_FLUENT_BIT_VERSION} ${AWS_FOR_FLUENT_BIT_VERSION}
+		create_manifest_list_init_debug ${1} "init-debug-latest" ${AWS_FOR_FLUENT_BIT_VERSION}
```

Comment:
Should we use $init here? I'm fine if you don't. This would just be for consistancy

Reply:
I don't understand why we added $init in the first place... let's not increase its usage